samples/Mvc.Server/Controllers/AuthorizationController.cs

```diff
@@ -101,7 +104,9 @@ public class AuthorizationController : Controller {
             var request = HttpContext.GetOpenIdConnectRequest();
 
             return View(new LogoutViewModel {
-                Parameters = request.Parameters
+                Parameters = request.ToDictionary(
```

Comment:
@ilmax note that if we agree to replace the entire OIDC payload by a simple request_id, we'll have to update the logout endpoint to make the same request_id conversion as the authorization endpoint.